oci/spec_opts_linux_test.go

```diff
@@ -510,3 +511,82 @@ daemon:x:2:root,bin,daemon
 		})
 	}
 }
+func TestWithLinuxDeviceFollowSymlinks(t *testing.T) {
+
+	// Create symlink to /dev/zero for the symlink test case
+	zero := "/dev/zero"
+	if _, err := os.Stat(zero); err != nil {
+		t.Fatal("Host does not have /dev/zero")
+	}
+
```

Comment:
This test is already using `assert` and `require`, please use it in the setup code for consistency too.  (Here: `require.NoError(t, err, "Host does not have /dev/zero")`)